packages/cli/commands/project/upload.js

```diff
@@ -59,9 +59,9 @@ exports.handler = async options => {
       message
     );
 
-    if (result.error) {
+    if (result.uploadError) {
```

Comment:
To check, does this reflect changes on the BE?

Reply:
Nevermind, just saw the changes you made in [lib/projects.js](https://github.com/HubSpot/hubspot-cli/pull/877/files#diff-fa4fa15b34bd2259fc2c24ec990fdbaa6492e6030d940700895318b399058174)